error_reporting/tests/system.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.cloud import error_reporting
+
+
+class TestErrorReporting(unittest.TestCase):
+
+    def test_report_exception(self):
+        client = error_reporting.Client()
+        try:
+            raise NameError
+        except NameError:
+            client.report_exception()
```

Comment:
Yes, I can use the gax client to do it, though that makes it so the test will only work with gRPC (unless I added an HTTP client only for the purpose of a system test, the current non-gRPC approach uses Logging which has no means of getting the errors).

I didn't add it yet in the spirit of small PRs, can add it on this one or a new one (sounds like you prefer this one).